ncl/keymap-codegen.ncl

```diff
@@ -295,6 +296,25 @@ let validators = import "validators.ncl" in
               rust_expr = "crate::key::layered::ModifierKey::hold(%{std.to_string layer_index})",
               json_value = k,
             },
+          { SetActiveLayers = active_layer_indices } =>
+            {
+              key_type = {
+                key_type = "crate::key::layered::ModifierKey",
+                as_rust_type_path = key_type,
+              },
+              key_impl = {
+                base = key_type.key_type,
+              },
+              rust_expr =
+                let comma_separated_indices =
+                  active_layer_indices
+                  |> std.array.map std.to_string
+                  |> std.string.join ", "
+                in
+                let active_indices_expr = "&[%{comma_separated_indices}]" in
+                "crate::key::layered::ModifierKey::set_active_layers(%{active_indices_expr})",
+              json_value = k,
+            },
```

Comment:
It's a nice thought; but, using the serde `Derive`'d using a bitset as the value for the modifier key variant, then the "json" value *must* be the bitset also.

(Or, explicitly implement Derive).